sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/DefaultSynchronousMetricStorage.java

```diff
@@ -89,6 +89,16 @@ public void recordLong(long value, Attributes attributes, Context context) {
 
   @Override
   public void recordDouble(double value, Attributes attributes, Context context) {
+    if (Double.isNaN(value)) {
+      logger.log(
+          Level.WARNING,
```

Comment:
I'd prefer `DEBUG` for these (I see a good number of support cases generated by concern over WARNING/ERROR messages)

Reply:
👍